qt/python/mantidqt/widgets/colorbar/colorbar.py

```diff
@@ -161,15 +177,20 @@ def norm_changed(self):
         Called when a different normalization is selected
         """
         idx = self.norm.currentIndex()
+        scale = NORM_OPTS[idx]
```

Comment:
Could you help me understand where `scale` is used now? I can't see it used here anymore.

Reply:
Ah it's not, I thought I'd removed this